### DISCUSSION TITLE
Send an image in the notification payload

### DISCUSSION THREAD

**Discussion by rizalahmaddd** [lkaybob/php-fcm-v1#16] [Q&A] [answered]
how to send an image in the notification payload? 
thanks

**Comment by lkaybob** [comment, marked as answer]
Currently attaching image on notification payload is not supported with php-fcm-v1. It looks like HTTP v1 API does support it, so it needs to be implemented in php-fcm-v1

https://firebase.google.com/docs/cloud-messaging/android/send-image?hl=ko#rest

**Comment by ManuxGomez** [comment]
Hello @lkaybob  I made a few modifications to your code and that allows me to send the images. I would like to be able to add my participation with this code fragment to your project, will it be possible?

**Reply by lkaybob** [reply]
Thank you in advance, and I certainly welcome your contribution!

Along with the modification, I recommend you to add test case for the modification as well.
After that, please create a pull request so that I can review and merge it.

**Reply by mgcastelberg** [reply]
Hello @lkaybob  It's me again Manuel Gomez, do you think I can upload a branch with the changes for you to review?

I get the following error when trying to upload the branch: remote: Permission to lkaybob/php-fcm-v1.git denied.

I already added the unit and integration tests to add an image for Android notifications.

Could you send me the invitation for this user, the other one is from the company where I work, and I prefer to contribute with my developer profile @mgcastelberg.

**Reply by lkaybob** [reply]
Please fork this repo and push your commit there, and then create PR with the target branch pointing to original repo's master branch.

You can check detailed instructions from the articles below.

https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks

https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork
